Formula/o/opentap.rb

```diff
@@ -0,0 +1,83 @@
+class Opentap < Formula
+  desc "Fast and easy development and execution of automated tests"
+  homepage "https://opentap.io"
+  url "https://github.com/opentap/opentap.git",
+    tag:      "v9.24.2",
+    revision: "bafe5eb48b0bcad2bd0f07198b69b6a58608807b"
+  license "MPL-2.0"
+
+  depends_on "dotnet"
+
+  uses_from_macos "unzip" => :build
+
+  def install
+    full_version = "9.24.0+2569356e"
+    os = OS.mac? ? "MacOS" : "Linux"
+    arch = Hardware::CPU.intel? ? "x64" : "arm64"
+    package_output_path = buildpath/"OpenTAP.TapPackage"
+    dotnet = Formula["dotnet"]
+    args = %W[
+      --configuration Release
+      /p:AssemblyVersion=#{version}
+      /p:FileVersion=#{version}
+      /p:InformationalVersion=#{version}
+      /p:Version=#{version}
+    ]
+
+    # Build OpenTAP
+    system "dotnet", "build", "tap/tap.csproj", *args
+    system "dotnet", "build", "Cli/Tap.Cli.csproj", *args
+    system "dotnet", "build", "Package/Tap.Package.csproj", *args
+    system "dotnet", "build", "BasicSteps/Tap.Plugins.BasicSteps.csproj", *args
+
+    # Replace the version in the package.xml file
+    inreplace "package.xml", "$(GitVersion)", full_version.to_s
```

Comment:
can you use version instead of hardcoded full_version?

Reply:
Yes, but this would `version` does not have the sha, thereby removing the "+4324232". This is not strictly necessary, but is something we would prefer having.